.travis.yml

```diff
@@ -19,8 +19,8 @@ before_script:
   - composer install -n
 
 script:
-  - if [[ "$ANALYSIS" != 'true' ]]; then vendor/bin/phpunit ; fi
-  - if [[ "$ANALYSIS" == 'true' ]]; then vendor/bin/phpunit --coverage-clover clover.xml ; fi
+  - if [[ "$ANALYSIS" != 'true' ]]; then vendor/bin/phpunit -dxdebug.mode=coverage; fi
+  - if [[ "$ANALYSIS" == 'true' ]]; then vendor/bin/phpunit -dxdebug.mode=coverage --coverage-clover clover.xml ; fi
```

Comment:
Build is failing on `7.3` because of this 🤔

Reply:
It's related to Xdebug 3.0. Turns out that `XDEBUG_MODE=coverage ./vendor/bin/phpunit …` is the correct format!

Reply:
Rebased out incorrect versions.